src/python/espressomd/shapes.py

```diff
@@ -18,7 +18,13 @@
 
 
 @script_interface_register
-class Cylinder(ScriptInterfaceHelper):
+class Shape(ScriptInterfaceHelper):
```

Comment:
I don't think that Shape should be a script interface object. Just use multiple inheritance, don't use a base class here and don't set `_so_name` here. `Shapes::Shape` does not exist and this class can not be instantiated anyway.

Reply:
Thanks!